src/main/java/net/neoforged/neoforge/event/level/BlockEvent.java

```diff
@@ -99,6 +99,49 @@ public void setExpToDrop(int exp) {
         }
     }
 
+    /**
+     * Fired when a block is destroyed AFTER drops have been determined, but have not yet spawned.
+     * - Cancellation will result in the drops not being spawned.
+     * - Breaker may be null if the block was not destroyed by an entity.
+     * - The drops list is immutable and cannot be modified. Use LootTableModifiers instead
+     * - It is safe to modify the Block in this event, as it has already been replaced.
+     */
+    public static class DestroyedEvent extends BlockEvent implements ICancellableEvent {
```

Comment:
The event should be renamed to `BlockDropsEvent`. 
1. The current name is too similar to `BreakEvent`
2. The action of the event concerns the harvesting, not the destruction. Cancelling the event prevents the harvest of the drops.
3. `BlockDropsEvent` promotes consistency with the other event of the same purpose, `LivingDropsEvent`.

Reply:
I agree, it has been renamed.